content/02.introduction.md

```diff
@@ -23,7 +23,7 @@ We do not focus on Git in this paper, but we recommend users explore the many re
 
 <!-- ### General background on how GitHub is used by EEB researchers -->
 <!-- *Contributors to this section: RCO, PHPB* -->
-The voluminous user-community and the numerous resources providing pedagogical instruction material on how to use GitHub streamlined its widespread use (CITE).
+The expansive user-community and the numerous resources providing pedagogical instruction material on how to use GitHub streamlined its widespread use [@doi:10.1371/journal.pcbi.1004947; @doi:10.1080/00031305.2017.1399928, @https://happygitwithr.com/, @https://ourcodingclub.github.io].
 Nevertheless, although multiple articles have encouraged researchers in EEB to adopt GitHub as part of their research process [@doi:10.1038/s41559-017-0160; @doi:10.1038/538127a], its use is still not widespread.
 Because GitHub and its features have been centered on collaboration for software development in information systems [@doi:10.1109/ASONAM.2016.7752419], first-time users from domains without formal training in information technology may face steep learning curves.
```

Comment:
Line 28 (02.introduction.md)- this is very old fashioned of me but we can't start a sentence with "Because"... maybe "This is because..."

Reply:
The usage of because here to start the sentence is from the rearranging of clauses within the sentence, so it's still a complete sentence. I'll put them back in the opposite order!